rochester_security_summit.md

```diff
@@ -252,15 +252,15 @@ Schedule queries with "packs"
 
 # Loki
 
-> Prometheus-inspired logging for cloud natives.
+> like Prometheus, but for logs.*
```

Comment:
I took this from https://grafana.com/oss/loki.

Reply:
lol you are better at this than I am :)

I don't think I like that tag line any better but that's a discussion I should have with someone else :)